### ISSUE TITLE
This is not really an image editor as the image doesn't get changed

### ISSUE THREAD

**Issue by codepo8** [lalwanivikas/image-editor#4]
You ask in the article for a feature to save the image once the filters were applied. As the pixels of the image don't get manipulated with CSS filters, I don't know how that would be possible. By moving the functionality to canvas, this would be easy.

**Comment by lalwanivikas** [comment]
I agree with you Christian :+1: . Canvas is the way to go if we want to save the modified image. I'll work on it and update you when it's ready. I am also sending you a personal mail. Thanks!

**Comment by lalwanivikas** [comment]
I agree with you @codepo8  :+1: . Canvas is the way to go if we want to save the modified image. I'll work on it and update you when it's ready. I am also sending you a mail. Thanks!

**Comment by impressivewebs** [comment]
Is it possible you could keep the CSS filters functionality and use something like this?

http://html2canvas.hertzen.com/

I'd like to see the CSS filters be used, and then some kind of screenshot API utilized that will take a screenshot of the image only (maybe defined using something like getBoundingClientRect() in JS). I don't know if this is possible, that's just a wild brainstorm.

**Comment by codepo8** [comment]
I'd be surprised if html2canvas supports that functionality. It'd have to be a tool that supports CSS filters and creates an image from the content after CSS was applied. Mozilla had a thing called MozElement for that (http://hacks.mozilla.org/2010/08/mozelement/) but it got removed because of security reasons.

**Comment by timothymarois** [comment]
If able to save the image after the filter, would be amazing. Any updates on doing so? Otherwise really good work so far!

**Comment by lalwanivikas** [comment]
@timothymarois naaa....sorry :(

But this might help you: http://camanjs.com/
